blocks/title/title.css

```diff
@@ -0,0 +1,125 @@
+.title {
+    /* The gradient here doesn't look great on desktop, better on mobile. Maybe we can create a slightly different one later */
+    background: var(--gradient);
+    width: 100vw;
+    margin-left: calc(-50vw + 50%);
+    padding: 26px;
+    border-top: 2px solid var(--blue);
+    border-bottom: 2px solid var(--blue);
+    position: relative;
+    margin-top: 80px;
+    margin-bottom: 40px;
+}
+
+.title h2 {
+    line-height: 1;
+    margin: 1px 0 0;
+    overflow-wrap: break-word;
+}
+
+@keyframes rotate-background {
+  from {
+        background-position: center center;
+        transform: rotate(0deg);
+    }
+
+  to {
+        background-position: center center;
+        transform: rotate(360deg);
+    }
+}
+
+.title .title-price {
+    position: absolute;
+    height: 130px;
+    width: 130px;
+    top: -40%;
+    right: 10%;
+    display: flex;
+    flex-flow: column;
+    justify-content: center;
+    align-items: center;
+    margin: 0;
+    overflow: hidden;
+}
+
+.title .title-price::before {
+    content: '';
+    position: absolute;
+    background-image: url('../../icons/pricetag.svg');
+    background-repeat: no-repeat;
+    background-size: contain;
+    background-position: center;
+    height: 100%;
+    width: 100%;
+    top: 0;
+    right: 0;
+    display: flex;
+    flex-flow: column;
+    justify-content: center;
+    align-items: center;
+    z-index: 1;
+    animation: rotate-background 10s linear infinite;
+    opacity: 1;
+}
+
+.title .title-price-content {
+    position: relative;
+    z-index: 2;
+    transform: rotate(-360deg);
+    display: flex;
+    flex-flow: column;
+    justify-content: center;
+    align-items: center;
+}
+
+.title .title-price-amount {
+    font-family: var(--heading-font-family);
+    font-size: var(--heading-size-xl);
+    text-align: center;
+    line-height: 34px;
+    padding-top: 10px;
+}
+
+.title .title-price-subtext {
+    text-align: center;
+    font-family: var(--heading-font-family);
+    font-size: var(--heading-size-xs);
+    line-height: 24px;
+}
+
+.title .title-price-long-subtext {
+    text-align: center;
+    font-family: var(--heading-font-family);
+    font-size: var(--heading-size-xxs);
+    line-height: 24px;
```

Comment:
most of this is duplicative. is there a way to style all the subtext in one declaration, then update the font size in one line in another?